packages/@glimmerx/core/tests/render-tests.ts

```diff
@@ -184,5 +184,65 @@ export default function renderTests(
       const html = await render(MyComponent);
       assert.strictEqual(html, 'Hello TEST', 'the component rendered');
     });
+
+    test('can render with a lazy owner and nested services', async (assert) => {
```

Comment:
Is this test needed? Seems to be basically the same functionality in the other test.

Reply:
We have two different render methods, `@glimmerx/core and `@glimmerx/ssr`. These render methods are simple, but use our new `Owner` implementation, and the respective modules use their  two different upstream libraries `@glimmer/core` and `@glimmer/ssr`.

So it's sort of overkill but it ensures we are using the upstream render methods correctly given the two different APIs.